cosmwasm/upgradable/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+#![warn(clippy::pedantic)]
+#![allow(clippy::missing_errors_doc)]
+#![cfg_attr(not(test), warn(clippy::unwrap_used))]
+
+use cosmwasm_std::{DepsMut, Env, Event, MessageInfo, Response, WasmMsg, to_json_binary};
+use frissitheto::UpgradeMsg;
+
+use crate::{error::ContractError, msg::ExecuteMsg};
+
+pub mod error;
+pub mod msg;
+
+#[cfg_attr(not(feature = "library"), cosmwasm_std::entry_point)]
```

Comment:
this contract has no way of being a contract by it's own right? I think we should make it a proper library, with no `execute` but proper functions (eg `fn upgrade`)